jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -1441,6 +1441,9 @@ void java_object_factoryt::gen_nondet_array_init(
   else
   {
     // Arrays of primitive types can be initialized with a single instruction
+    // We don't do this for arrays of Booleans, because Bools are represented
```

Comment:
These are `bool`s not `Boolean`s (i.e. we're talking about the primitive type)

Reply:
Oh yes, I wanted to sound smart by correctly capitalising Boolean, but then of course we can confuse it with Boolean objects. Good catch. Since this is blocking TG, I will leave it as it is and do a PR on top of that to correct the typo.